taca/deliver/deliver.py

```diff
@@ -0,0 +1,385 @@
+"""
+    Module for controlling deliveries of samples and projects
+"""
+import datetime
+import glob
+import hashlib
+import os
+import re
```

Comment:
I just wanted to say I noticed you listed these imports in alphabetical order and I love you for it.